pages/employer/index.tsx

```diff
@@ -8,8 +9,13 @@ import GeneralLayout from 'layouts/generalLayout';
 import { employerData } from "utils";
 import employerImg from "assets/general/employer-team.svg"
 import { CompHiringCard, EmployerSideBar } from '../../components/Employer';
-
+import { useRouter } from "next/router";
 const Employer: NextPage = () => {
+    const router = useRouter();
```

Comment:
please why is this so??
routing the employer page back home?

Reply:
yes they said we should do that because of our soft launch since the page isnt ready 
directives is from geo and the project manager 
@sammy-code98

Reply:
I think a better approach would have been to route the user to the page,  display a coming soon design to the user with a button to take them back home.
The (static) code for the page itself can be commented out.

Routing the user with a useEffect to home page each time they click on a link like this one is bad UX.